### PR TITLE
docs: Release notes for 0.13.8.

### PR DESCRIPTION
**Pull request by determined-dsw**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,128 @@
...
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
+
+-  Security: Allow the agent to use different addresses for master
```
Comment (neilconway) [inline docs/release-notes.txt:23]: I don't really follow this or understand why it is useful.

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,128 @@
...
+==============
+
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
```
Comment (neilconway) [inline docs/release-notes.txt:21]: Not sure how to actually use this feature -- the docs (`how-to/installation/aws.txt`) haven't been updated, we don't give example usage here, and the relationship between subnets and AZs isn't immediately clear to me.

Comment (neilconway) [reply docs/release-notes.txt:21]: cc @armandmcqueen

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,128 @@
...
+   certificate verification and connecting to the master.
+
+**Improvements**
+
+-  **Breaking Change:** API: Perform salting and hashing on server-side for
+   the password change endpoint. This makes this endpoint consistent with
+   the new login endpoint described at
+   https://docs.determined.ai/latest/rest-api/ .
+
+-  **Breaking Change:** Logging: Start using Fluent Bit for handling trial
```
Comment (neilconway) [inline docs/release-notes.txt:33]: What are the user-visible consequences of this change? @dzhu 

e.g., does this result in better scalability, more readable logs, more options for storing logs, ... ?

File: docs/release-notes.txt [hunk 4]
```diff
@@ -8,6 +8,128 @@
...
+
+-  API: Add support for models that subclass ``tf.keras.Model`` when 
+   using the Determined TFKerasTrial API.
+
+-  API: Add tf.keras DCGAN example that subclasses ``tf.keras.Model``.
+
+-  API: Add ``self.context.configure_fit()`` to the Determined
+   TFKerasTrial API. Please see :ref:`tf-keras-trial` for detail.
+
+-  Kubernetes: Add option to configure the service type of the Determined
```
Comment (neilconway) [inline docs/release-notes.txt:64]: What is this useful for? Can we give an example of a "service type" you might want to configure? @aaron276h

File: docs/release-notes.txt [hunk 5]
```diff
@@ -8,6 +8,128 @@
...
+
+-  API: Add ``self.context.wrap_optimizer()`` to the Determined
+   TFKerasTrial API. Please see :ref:`tf-keras-trial` for detail.
+
+-  API: Add support for models that subclass ``tf.keras.Model`` when 
+   using the Determined TFKerasTrial API.
+
+-  API: Add tf.keras DCGAN example that subclasses ``tf.keras.Model``.
+
+-  API: Add ``self.context.configure_fit()`` to the Determined
```
Comment (neilconway) [inline docs/release-notes.txt:61]: What does this actually enable? Can we give an example of what this is useful for? @rb-determined-ai

File: docs/release-notes.txt [hunk 6]
```diff
@@ -8,6 +8,128 @@
...
+   defaults to ``auto``. See 
+   :ref:`Agent configuration <agent-configuration>` for details.
+
+-  API: Add ``self.context.wrap_optimizer()`` to the Determined
+   TFKerasTrial API. Please see :ref:`tf-keras-trial` for detail.
+
+-  API: Add support for models that subclass ``tf.keras.Model`` when 
+   using the Determined TFKerasTrial API.
+
+-  API: Add tf.keras DCGAN example that subclasses ``tf.keras.Model``.
```
Comment (neilconway) [inline docs/release-notes.txt:59]: This commit (765275a0cb8ffda09ff01a0dfaa0955e6541a000) did not update `docs/examples/index.txt` for the new DCGAN example -- can you fix, @aaron276h ?

File: docs/release-notes.txt [hunk 7]
```diff
@@ -8,6 +8,128 @@
...
+   ``slot_type`` of the agent configuration ``agent.yaml``. Dynamic agents 
+   having GPUs will be configured to ``gpu`` while those agents having no 
+   GPUs will be configured to ``none``. For static agents this field 
+   defaults to ``auto``. See 
+   :ref:`Agent configuration <agent-configuration>` for details.
+
+-  API: Add ``self.context.wrap_optimizer()`` to the Determined
+   TFKerasTrial API. Please see :ref:`tf-keras-trial` for detail.
+
+-  API: Add support for models that subclass ``tf.keras.Model`` when
```
Comment (neilconway) [inline docs/release-notes.txt:56]: Can we add a bit more detail here on why this is useful or what you can do with it? Doesn't this qualify as a major feature we should add to the "New Features" list? @aaron276h

File: docs/release-notes.txt (hunk 5 above)
Comment (rb-determined-ai) [reply docs/release-notes.txt:61]: how about this:
```rest
-  API: Add ``self.context.configure_fit()`` to the Determined
   TFKerasTrial API. Many parameters which would be passed to
   ``model.fit()``, such as ``class_weight``, ``verbose``, or
   ``workers``, can now be passed to ``configure_fit()`` and will be
   honored by ``TFKerasTrial``.  Please see :ref:`tf-keras-trial` for
   detail.
```

File: docs/release-notes.txt (hunk 7 above)
Comment (aaron276h) [reply docs/release-notes.txt]: done

File: docs/release-notes.txt (hunk 6 above)
Comment (aaron276h) [reply docs/release-notes.txt:61]: Good call, updated this.

File: docs/release-notes.txt (hunk 4 above)
Comment (aaron276h) [reply docs/release-notes.txt]: done

File: docs/release-notes.txt (hunk 3 above)
Comment (dzhu) [reply docs/release-notes.txt:38]: Honestly, not much by itself (though it does come along with the filtering changes, which I'm going to add a line for separately). Do you think it's worth mentioning here how this is laying the groundwork for future things?

File: docs/release-notes.txt (hunk 1 above)
Comment (dzhu) [reply docs/release-notes.txt:23]: There's an expanded version coming from the release spreadsheet:

```
Allow the master to use a TLS certificate that is valid for a different name than the agents use to connect to it. This ability is useful in situations where the master is accessed using multiple different addresses (e.g., private and public IP addresses of a cloud instance).

The agent now accepts a ``--security-tls-master-cert-name`` option to override the expected name in the master's TLS certificate. The CLI uses the ``DET_MASTER_CERT_NAME`` environment variable for the same purpose.
```

File: docs/release-notes.txt (hunk 3 above)
Comment (neilconway) [reply docs/release-notes.txt:38]: Sure, I don't think that would hurt.

Comment (dzhu) [reply docs/release-notes.txt:38]: Okay, how about this (appended to what's there now)?
```
This change is part of an effort to improve the handling of trial logs by increasing scalability and allowing more options for log storage.
```

File: docs/release-notes.txt (hunk 2 above)
Comment (armandmcqueen) [reply docs/release-notes.txt:21]: How about:

When using the 'simple' deployment type with 'det deploy aws', you can now use the '--agent-subnet-id' flag to specify which existing subnet agents should be launched in. As each subnet is associated with a single availability zone, this allows users to explicitly choose an availability zone that has GPU instances (there is no public information about which availability zone has GPUs so trial and error is the suggested approach). 

I'm on my phone and can't find backticks, but 'det-deploy aws' and '--agent-subnet-id' should be formatted as code.
